ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -9125,8 +9110,11 @@ static int zend_jit_recv_init(dasm_State **Dst, const zend_op *opline, zend_op_a
 			|	LOAD_ZVAL_ADDR r0, res_addr
 			|	ZVAL_DEREF r0, MAY_BE_REF
 			if (!ZEND_TYPE_IS_CLASS(arg_info->type)) {
-			|	cmp byte [r0 + 8], ZEND_TYPE_CODE(arg_info->type)
-			|	jne >8
+			|	mov edx, 1
+			|	mov cl, byte [r0 + 8]
+			|	shl edx, cl
+			|	test edx, ZEND_TYPE_MASK(arg_info->type)
+			|	je >8
```

Comment:
It should be possible to use "bt" instruction. Add comment, I'll take care later.